interactions/api/models/message.py

```diff
@@ -1094,3 +1094,84 @@ def __init__(self, **kwargs):
 
         if self.footer:
             self._json.update({"footer": self.footer._json})
+
+    def add_field(self, name: str = None, value: str = None, inline: bool = True):
```

Comment:
```suggestion
    def add_field(self, name: str, value: str, inline: Optional[bool] = False):
```
Name and value are required arguments, they should not have a default value. 

Inline is not present by default and not required, so please keep it to `False`


Also please keep in mind to use the `Optional` typehint for not required values

Reply:
Oh yeah my bad for the name and value, however for the inline, I think it should be by default to True ?

Reply:
It is not present by default, so it is either `None` or `False`

Reply:
Ok, I was used to dpy defaulting to True.

Reply:
Set it to none then, so we don't send any value for it

Reply:
Ok